credentialdigger/client.py

```diff
@@ -1144,14 +1144,19 @@ def _scan(self, repo_url, scanner, models=None, force=False, debug=False,
         discoveries_ids = list()
         if debug:
             logger.debug('Update database with these discoveries.')
-            for i in tqdm(range(len(new_discoveries))):
-                curr_d = new_discoveries[i]
-                new_id = self.add_discovery(
-                    curr_d['file_name'], curr_d['commit_id'],
-                    curr_d['line_number'], curr_d['snippet'], repo_url,
-                    curr_d['rule_id'], curr_d['state'])
-                if new_id != -1 and curr_d['state'] != 'false_positive':
-                    discoveries_ids.append(new_id)
+            total_discoveries = len(new_discoveries)
+            with Progress() as progress:
+                inserting_task = progress.add_task("Inserting discoveries...",
+                                                   total=total_discoveries)
+                for i in range(total_discoveries):
```

Comment:
if we replace tqdm then we can also update this loop into a for-each (`for curr_discovery in new_discoveries`) since we don't need the index `i` anymore